dockstore-webservice/src/main/java/io/dockstore/webservice/resources/ResourceConstants.java

```diff
@@ -35,7 +35,7 @@ public final class ResourceConstants {
     public static final String CURATION = "List and modify notifications for users of Dockstore";
     public static final String NIHDATACOMMONS = "Needed for SmartAPI compatibility apparantly, might be cargo cult behaviour";
     public static final String LAMBDAEVENTS = "Query lambda events triggered by GitHub Apps";
-    public static final String OPENAPI_JWT_SECURITY_DEFINITION_NAME = "bearer";
+    public static final String OPENAPI_JWT_SECURITY_DEFINITION_NAME = "BEARER";
```

Comment:
Oh, I see that this is an existing constant, so the webservice has two constants for `BEARER`. Your call if you wanna just use one constant globally :slightly_smiling_face:

Reply:
@kathy-t yeah I was confused about why there were two present, I'll consolidate it down.

Reply:
@kathy-t I opted to remove `io.dockstore.webservice.Constants.JWT_SECURITY_DEFINITION_NAME` and consolidate to `io.dockstore.webservice.resources.ResourceConstants.JWT_SECURITY_DEFINITION_NAME`
in the fixup commit.